test/lib/config/browser-options.js

```
@@ -310,6 +310,62 @@ describe('config browser-options', () => {
         });
     });
 
+    describe('screenshotsDir', () => {
+        it('should set default screenshotsDir option if it does not set in config', () => {
+            const config = createConfig();
+
+            assert.equal(config.screenshotsDir, defaults.screenshotsDir);
+        });
+
+        it('should throw error if screenshotsDir is not a string or function', () => {
```

should throw an error if a value